session/manager.go

```
@@ -0,0 +1,883 @@
+			return "", statErr
+		}
+		parent := filepath.Dir(current)
+		if parent == current {
+			return "", os.ErrNotExist
+		}
+		missing = append(missing, filepath.Base(current))
+		current = parent
+	}
+}
+
+func (s *Session) usage() (int64, error) {
```

golangci-lint (2.12.2)

[error] func (*Session).usage is unused

(unused)